app/src/main/java/in/tosc/digitaloceanapp/activities/DetailDropletActivity.java

```diff
@@ -87,8 +90,34 @@ public boolean onCreateOptionsMenu(Menu menu) {
     public boolean onOptionsItemSelected(MenuItem item) {
         int id = item.getItemId();
         if (id == R.id.delete_droplet) {
-
             // TODO: 26/11/16 perform delete
+            new MaterialDialog.Builder(this)
+                    .title(R.string.delete_droplet)
+                    .content("")
```

Comment:
Add some information
(Maybe some warning like - "This action is not reversible" in the content